lib/reader.js

```diff
@@ -39,8 +39,10 @@ module.exports = async (feeds, options) => {
     await Promise.all(
         feeds.map(async feed => {
             const root = join(opts.directory, await hashFeed(feed));
-            const entrypoint = feedEntrypoint(feed);
-            entries.push(join(root, entrypoint.file));
+            const entrypoints = feedEntrypoints(feed);
```

Comment:
I'd do 
```js
feedEntrypoints(feed).forEach(entrypoint => entries.push(join(root, entrypoint.file)));
```

but doesn't really matter.

Reply:
Cool, yea could do. Think i'll just plow ahead for now though.

Reply:
or even just inline the `feed-entrypoints.js` file and do
```js
feed
  .filter(item => Boolean(item.entry))
  .map(entrypoint => join(root, entrypoint.file))
  .forEach(file => entries.push(file));
```